test/torch/hook/test_hook_args.py

```diff
@@ -1,18 +1,111 @@
 from syft.frameworks.torch.hook import hook_args
 from syft.generic import pointers
-import torch
+import torch as th
+import torch.nn as nn
+import torch.optim as optim
 import numpy as np
 
 
 def test_build_rule_syft_tensors_and_pointers():
     pointer = pointers.PointerTensor(
         id=1000, location="location", owner="owner", garbage_collect_data=False
     )
-    result = hook_args.build_rule(([torch.tensor([1, 2]), pointer], 42))
+    result = hook_args.build_rule(([th.tensor([1, 2]), pointer], 42))
     assert result == ([1, 1], 0)
 
 
 def test_build_rule_numpy():
     arr = np.array([2.0, 3.0, 4.0])
     result = hook_args.build_rule([arr, arr + 2, [2, 4, "string"]])
     assert result == [1, 1, [0, 0, 0]]
+
+
+def test_backward_multiple_use(workers):
```

Comment:
why are there no assertions in this test? how do we know backwards actually worked?

Reply:
I'm only interested in having backward not failing due to hook_args, so I didn't checked that it works correctly because it's checked elsewhere
But I agree this test leaves both a feeling of deja vu and incomplteness

Reply:
Ok @robert-wagner this is addressed!